service_extensions/callouts/add_header/protos/service.proto

```diff
@@ -0,0 +1,639 @@
+syntax = "proto3";
```

Comment:
suggestion: please consider publishing the protofile using official path used by client libraries (I will share details over chat)